WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
"Attilid" is also pure conjecture and is nowhere as bad-A as House the Hun.

Unlike Genghis Khan, we do not know the true last name of Attila the Hun. Both "the Hun" and "Scourge of God" were ancient nicknames and both are fitting.